xml_converter/generators/code_generator.py

```diff
@@ -393,11 +405,10 @@ def write_attribute(self, output_directory: str) -> None:
         file_loader = FileSystemLoader('cpp_templates')
         env = Environment(loader=file_loader)
         attribute_names: Dict[str, str] = {}
+        template_variables: List[Tuple[str, str, str, List[str]]] = []
```

Comment:
Same thing with renaming the tuple instead of changing it to a dataclass

Reply:
Changed to dataclass `AttributeVariable`